script.js

```diff
@@ -54,20 +54,19 @@ function displayFinalMessage() {
     let p2Score = playerTwoScore;
     let finalWinnerMessageElement = document.querySelector(".final-winner");
     let finalWinner = "";
+    let allButtons = document.querySelectorAll(".content");
```

Comment:
Little strange that the class selector to get all buttons is "content"? 

It'd be nicer if you just selected all buttons by tag or a more specific class